### PR TITLE
View training grouped by category for a specific role (#47)

### PR DESCRIPTION
**Pull request by clarebuckley**

### CODE REVIEW

File: e2e/src/app.e2e-spec.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import { LandingPage } from './landing-page.po';
 import { BandDetailViewerPage } from './band-detail-viewer-page.po';
+import { RoleDetailViewerPage } from './role-detail-viewer-page.po';
 import { browser, logging, protractor } from 'protractor';
 
 describe('Landing Page', () => {
   let page: LandingPage;
-  let bandDetailPage: BandDetailViewerPage
+  let bandDetailPage: BandDetailViewerPage;
```
Comment (micalexis) [inline e2e/src/app.e2e-spec.ts:8]: where are these being used?

File: src/app/detail-viewer/detail-viewer.component.ts [hunk 2]
```diff
@@ -33,23 +33,38 @@ export class DetailViewerComponent implements OnInit {
...
-                     }
-                 }
+                 this.populateTrainingResources(response);
             });
+        }
+    }
 
+    //Populate variables for training resources and categories to be displayed using response from API
+    private populateTrainingResources(response: object){
+        if(response != null){
```
Comment (micalexis) [inline src/app/detail-viewer/detail-viewer.component.ts:53]:
```JAVASCRIPT
if(response){
}
```

File: src/backend-api/index.js [hunk 3]
```diff
@@ -72,6 +72,13 @@ app.get('/roles/:id', (req,res) => {
     });
 });
 
+app.get('/roles/:id/training', (req,res) => {
+    db.getRoleTraining(req.params.id, (err,rows) => {
+        if(err) return handleError(err);
```
Comment (micalexis) [inline src/backend-api/index.js:77]: put in brackets

File: src/backend-api/index.js [hunk 4]
```diff
@@ -72,6 +72,13 @@ app.get('/roles/:id', (req,res) => {
...
+        res.send(rows);
+    });
+});
+
 app.get('/capabilities/:id', (req,res) => {
     db.getCapability(req.params.id, (err,row) => {
         if(err) return handleError(err);
```
Comment (micalexis) [inline src/backend-api/index.js:84]: same as above

**Review by micalexis — CHANGES_REQUESTED** [review]

**Comment by clarebuckley** [comment]
Closed due to renaming branch to feature/view-role

**Pull request closed without merging**